README.markdown

```diff
@@ -278,6 +280,11 @@ Yichun "agentzh" Zhang (章亦春) <agentzh@gmail.com>, OpenResty Inc.
 
 [Back to TOC](#table-of-contents)
 
+Contributors
+===========
+
+[Zijian Wang (Herman)](https://github.com/actini) <actini@outlook.com>
```

Comment:
we usually do not maintain the contributors in README since GitHub already have a page for it:
https://github.com/openresty/lua-resty-upstream-healthcheck/graphs/contributors